atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/Schema.java

```diff
@@ -407,4 +422,37 @@ public void addCleanupTask(String rawTableName, Supplier<OnCleanupTask> task) {
     public void ignoreTableNameLengthChecks() {
         ignoreTableNameLengthChecks = true;
     }
+
+    public SchemaMetadata getSchemaMetadata() {
```

Comment:
This seems to be only for tests -- make it package-private and annotate as visible for tests?

Reply:
This is only for tests for now, but will be needed in Part 3 when we actually write the schema metadata to the KVS.